src/mono/mono.proj

```diff
@@ -481,7 +677,11 @@
   <Target Name="BuildMonoRuntimeUnix" Condition="'$(OS)' != 'Windows_NT'" DependsOnTargets="ConfigureMonoRuntimeUnix">
     <Message Text="--- Building Mono ---" Importance="High" />
     <Exec Command="make -j$([System.Environment]::ProcessorCount)" IgnoreStandardErrorWarningFormat="true" WorkingDirectory="$(MonoObjDir)" />
-    <Exec Condition="'$(TargetsiOS)' == 'true' or '$(TargetsAndroid)' == 'true'" Command="make install -j$([System.Environment]::ProcessorCount)" IgnoreStandardErrorWarningFormat="true" WorkingDirectory="$(MonoObjDir)" />
+    <Exec Condition="'$(TargetstvOS)' == 'true' or '$(TargetsiOS)' == 'true' or '$(TargetsAndroid)' == 'true'" Command="make install -j$([System.Environment]::ProcessorCount)" IgnoreStandardErrorWarningFormat="true" WorkingDirectory="$(MonoObjDir)" />
+
+    <Message Condition="'$(TargetstvOS)' == 'true' and '$(TargetstvOSSimulator)' != 'true'" Text="--- Building Mono AOT cross-compiler ---" Importance="High" />
+    <Exec Condition="'$(TargetstvOS)' == 'true' and '$(TargetstvOSSimulator)' != 'true'" Command="make -j$([System.Environment]::ProcessorCount)" IgnoreStandardErrorWarningFormat="true" WorkingDirectory="$(MonoObjDir)cross" />
+    <Exec Condition="'$(TargetstvOS)' == 'true' and '$(TargetstvOSSimulator)' != 'true'" Command="make install -j$([System.Environment]::ProcessorCount)" IgnoreStandardErrorWarningFormat="true" WorkingDirectory="$(MonoObjDir)cross" />
```

Comment:
This is the same as the block below for iOS. I think we should introduce a new property `_BuildMonoAOTCrossCompiler` or something like that in the target here and move the checks there:

```xml
<PropertyGroup>
  <_BuildMonoAOTCrossCompiler Condition="'$(TargetsiOS)' == 'true' and '$(TargetsiOSSimulator)' != 'true'">true</_BuildMonoAOTCrossCompiler>
  <_BuildMonoAOTCrossCompiler Condition="'$(TargetstvOS)' == 'true' and '$(TargetstvOSSimulator)' != 'true'">true</_BuildMonoAOTCrossCompiler>
</PropertyGroup>
```